### DISCUSSION TITLE
Claims inside Firebase Token

### DISCUSSION THREAD

**Discussion by dajevtic** [serverpod/serverpod#520] [Q&A] [answered]
Not posting the code here as I have fixed it in the server side, but I do have a question regarding the intention of scope field inside of "serverpod_user_info" table...
The firebase token I pass to authenticate towards serverpod server have claims about the user's role, but they are ignored on servpod side. Same happens if I store the claims inside the database field "scopeNames"...
Please, can you tell me if there is any implementation or future plan that is intended to store a "group", "claim" or "scope" of a user that logs in via firebase?

**Comment by vlidholt** [comment, marked as answer]
The scopes in Serverpod are not the same as the claims/scopes in Firebase or Google sign in.

Serverpod's scopes are used to restrict access to endpoints. Here is some information on how they work: https://docs.serverpod.dev/concepts/authentication#restricting-access-to-endpoints
